modules/tinymce/src/core/main/ts/newline/NewLineAction.ts

```diff
@@ -66,8 +66,7 @@ const hasShiftKey = (_editor: Editor, shiftKey: boolean) => {
 const canInsertIntoEditableRoot = (editor: Editor) => {
   const forcedRootBlock = Options.getForcedRootBlock(editor);
   const rootEditable = NewLineUtils.getEditableRoot(editor.dom, editor.selection.getStart());
-
-  return rootEditable && editor.schema.isValidChild(rootEditable.nodeName, forcedRootBlock);
+  return rootEditable.map((rE) => editor.schema.isValidChild(rE.nodeName, forcedRootBlock)).getOr(false);
```

Comment:
Rather than mapping to a boolean, I prefer filter. `x.map(y).getOr(false)` should be equivalent to `x.filter(y).isSome()`

Does that make sense?

Reply:
Yep, fixing now.